### ISSUE TITLE
Notifications in home app only shows up when home app open

### ISSUE THREAD

**Issue by ajghb** [home-assistant/homebridge-homeassistant#300]
i’m having a strange problems with notifications within ios / home app.

Notifications would previously show up on the locked screen and when swiping down etc even if home app was not open or even from my apple watch etc.

Now, those same notifications would pop up (irrespective of when the notification event actually happened) WHEN I launch and open the home app on ios side. If home app is never opened there would be no notifications at all.

Homebridge running on macos with the latest updates and ios latest released version.

My current setup is 

home-assistant —> homebridge —> homekit / home app.

**Comment by schmittx** [comment]
Have you confirmed that background app refresh is enabled within iOS?

**Comment by ajghb** [comment]
Confirmed that background app refresh is enabled. 

Further information: notifications was working fine until recently. The only two things I have done recently is 
1/ update HomeAssistant to 0.63.2 which caused some problems on the front end of homesssistant. I then downgraded back to 0.62.1. I didn’t think that was relevant but just sharing for completeness. 
2/ As part of trying to diagnose the problem I had deleted homebridge from the home app, deleted all the files in the persist directory in homebridge and started from scratch to add back homebridge devices into home app. 

And I also finally upgraded iOS to 11.2.5 in the last week or so. 

I can’t be sure if any of these things contributed or not to the current issue or if they are entirely unrelated as I did not realise the issue at first and not sure exactly of the cause and effect sequence (if at or).

Any other thoughts or areas to investigate would be appreciated.
